### PR TITLE
Add ParadeDB

### PR DESCRIPTION
**Pull request by philippemnoel**
Nice chatting, and great project. Excited to add ParadeDB

### CODE REVIEW

**Comment by philippemnoel** [comment]
Thanks for fixing the PR, lgtm!

File: docs/tools/vdb_table/data/paradedb.json [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+  },
+  "geo_search": {
+    "support": "full",
+    "source_url": "https://github.com/orgs/paradedb/discussions/1110",
+    "comment": "PostGIS"
+  },
+  "multi_vec": {
+    "support": "full",
+    "source_url": "https://docs.paradedb.com/search/dense/overview",
+    "comment": ""
```
Comment (prrao87) [inline docs/tools/vdb_table/data/paradedb.json:52]: This docs page doesn't seem to describe the multi-vector search feature - does ParadeDB support storing and querying multiple vectors per document?

File: docs/tools/vdb_table/data/paradedb.json [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+  },
+  "facets": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "geo_search": {
+    "support": "full",
+    "source_url": "https://github.com/orgs/paradedb/discussions/1110",
+    "comment": "PostGIS"
```
Comment (prrao87) [inline docs/tools/vdb_table/data/paradedb.json:47]:
```suggestion
    "comment": "Available via the PostGIS extension"
```

File: docs/tools/vdb_table/data/paradedb.json [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+  },
+  "vector_launch_year": 2023,
+  "metadata_filter": {
+    "support": "full",
+    "source_url": "",
+    "comment": ""
+  },
+  "hybrid_search": {
+    "support": "full",
+    "source_url": "https://github.com/paradedb/paradedb",
```
Comment (prrao87) [inline docs/tools/vdb_table/data/paradedb.json:36]:
```suggestion
    "source_url": "https://docs.paradedb.com/search/hybrid/basic",
```

File: docs/tools/vdb_table/data/paradedb.json [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+    "comment": "https://docs.paradedb.com/search/hybrid/basic"
```
Comment (prrao87) [inline docs/tools/vdb_table/data/paradedb.json:37]:
```suggestion
    "comment": ""
```

File: docs/tools/vdb_table/data/paradedb.json [hunk 5]
```diff
@@ -0,0 +1,190 @@
...
+  "doc_size": {
+    "bytes": 0,
+    "unlimited": false,
+    "source_url": "",
+    "comment": ""
+  },
+  "vector_dims": {
+    "value": 2000,
+    "unlimited": false,
+    "source_url": "",
```
Comment (prrao87) [inline docs/tools/vdb_table/data/paradedb.json:148]:
```suggestion
    "source_url": "https://docs.paradedb.com/search/dense/index",
```

**Review by prrao87 — CHANGES_REQUESTED** [review]
Just a few small things related to source URLs.

File: docs/tools/vdb_table/data/paradedb.json (hunk 1 above)
Comment (prrao87) [reply docs/tools/vdb_table/data/paradedb.json:52]: I think this should be removed and marked as "none" - @dhruv-anand-aintech could you please confirm?

Comment (dhruv-anand-aintech) [reply docs/tools/vdb_table/data/paradedb.json:52]: yeah, I'll do that. @philippemnoel you can create a new PR with the appropriate docs url listed as source when you see this.

**Pull request merged**